### PR TITLE
[JENKINS-38276] text area width

### PR DESCRIPTION
**Pull request by ssbarnea**
# Description

See [JENKINS-38276](https://issues.jenkins-ci.org/browse/JENKINS-38276).

Details: TODO

### Changelog entries

Proposed changelog entries:

* Entry 1: JENKINS-38276, Increased width of pipeline editor

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Link to JIRA ticket in description, if appropriate
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For new API and extension points: Link to the reference implementation in open-source (or example in Javadoc)

### Desired reviewers

@mention

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -419,7 +419,7 @@ THE SOFTWARE.
     <dependency><!-- groovy shell uses this but it doesn't declare this dependency -->
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>1.11</version>
+      <version>1.14</version>
```
Comment (recena) [inline core/pom.xml]: unrelated?

Comment (ssbarnea) [inline core/pom.xml]: @recena Indeed, unrelated, even if it would not be a bad idea to upgrade it but clearly no need to include it here. I will mark it as [wip] as I still want to test it more, I may need or want to include few more layout fixes to improve the UX.

Comment (oleg-nenashev) [inline core/pom.xml]: @ssbarnea I would recommend to do the upgrade in a separate PR. Commonly in such PRs we provide links to available changelogs and the full code diff.

**Comment by daniel-beck** [comment]
WIPing this based on opening comment.

**Comment by daniel-beck** [comment]
@ssbarnea Do you plan to finish this?

**Comment by biancamrcf** [comment]
Thanks for reminding me, I will give it another go next week. The reality is that I fogot about it.

**Comment by ssbarnea** [comment]
I will work on it next week, I am really waiting for my flight now. Yep, forgot about this one, added a reminder.

**Comment by oleg-nenashev** [comment]
Basically there is no outstanding comments AFAICT. @ssbarnea are you still interested in getting it landed?

**Comment by jglick** [comment]
This is a much more general change than the specific problem mentioned in [JENKINS-38276](https://issues.jenkins-ci.org/browse/JENKINS-38276), and seems like it could cause regressions elsewhere. Also it would only seem to address horizontal space issues (and those only partially), and not for example make the ACE editor vertically resizable, which seems to be a more critical issue.
